cea/interfaces/dashboard/api/databases.py

```
@@ -7,6 +10,7 @@
 
 import cea.inputlocator
```

Ruff (0.8.2)

`cea.inputlocator` imported but unused

Remove unused import: `cea.inputlocator`

(F401)